extensions/twitch/package.json

```diff
@@ -0,0 +1,101 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "twitch",
+  "title": "Twitch",
+  "description": "Search for Streams, Channels or Games on Twitch",
+  "icon": "icon.png",
+  "author": "dieserRobin",
+  "license": "MIT",
+  "preferences": [
+    {
+      "name": "clientId",
+      "type": "password",
+      "default": "",
+      "title": "Client ID",
+      "required": true,
+      "description": "Twitch ClientId https://twitchapps.com/tokengen/"
+    },
+    {
+      "name": "authorization",
+      "type": "password",
+      "default": "",
+      "title": "Authorization Token",
+      "required": true,
+      "description": "Twitch Bearer Token (Must be matching with ClientId) https://twitchapps.com/tokengen/"
+    },
+    {
+      "name": "streamlink",
+      "type": "textfield",
+      "default": "/opt/homebrew/bin/streamlink",
+      "title": "Location of Streamlink binary",
+      "required": false,
+      "description": "Use 'where streamlink' or install via brew install streamlink"
+    },
+    {
+      "name": "lowlatency",
+      "type": "checkbox",
+      "label": "Low Latency",
+      "default": false,
+      "title": "Latency (VLC Player is required if enabled)",
+      "required": false,
+      "description": "VLC Player is required for low latency streams"
+    },
+    {
+      "name": "quality",
+      "type": "textfield",
+      "default": "best",
+      "title": "Video Quality",
+      "required": false,
+      "description": "Standard video quality"
+    }
```

Comment:
```suggestion
    {
      "name": "quality",
      "type": "dropdown",
      "default": "best",
      "title": "Video Quality",
      "required": false,
      "description": "Standard video quality",
	  "data": [{"title":  "Best", "value":  "best"},{"title":  "720p", "value":  "720p"},{"title":  "480p", "value":  "480p"},{"title":  "Worst", "value":  "worst"}]
    }
```
Could it be an idea to have the quality setting something like this?

![DF6N4rT9HjuNht9K](https://user-images.githubusercontent.com/38033107/148107676-8e80f987-3768-44a8-8b33-58cecd294912.png)

Reply:
Thanks for that suggestion! You're right a dropdown for the quality is much more user friendly

Reply:
@dieserRobin not only friendly, but I guess you can take this for your life, but: **EVERYTHING** (read this out loud few times) you could do to make the user life easier and error-free, you MUST do. Software means that you should never trust what kind of data it’s going to be inputted/received, you must cover almost every corner you can. Per’s suggestion is not only friendly or to improve the UX, but also to allow you to avoid to deal with/manage unnecessary data.